src/generators/image_generators/stable_diffusion_with_loras.py

```diff
@@ -164,7 +177,7 @@ def request_character_image_generation(
             tags=[],  # no tags, as this shouldn't be used in chathistory for anything else (at the moment)
         )
 
-        task.wait()
+        task.wait(retry_delay_s=0.1)
```

Comment:
thought: should we move the `task.wait(...)` into `self.generate()` so that if we need to change this again, there is a single place for it?

Reply:
👍